examples/glium_gl_area/main.rs

```diff
@@ -0,0 +1,40 @@
+use std::ptr;
+
+use gtk::prelude::*;
+
+mod glium_gl_area;
+use glium_gl_area::GliumGLArea;
+
+fn main() {
+    // Load GL pointers from epoxy (GL context management library used by GTK)
+    {
+        #[cfg(unix)]
+        let library = unsafe { libloading::os::unix::Library::new("libepoxy.so.0") }.unwrap();
```

Comment:
This won't work on macOS (`libepoxy.0.dylib` AFAIU)

Reply:
I don't have macOS so I can't check if it works either way..

Reply:
Maybe just add a case for macOS with that and then someone on macOS can worry about it :)